packages/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemModule.java

```diff
@@ -325,27 +342,23 @@ public void writeAsStringAsync(String uriStr, String string, Map<String, Object>
     try {
       Uri uri = Uri.parse(uriStr);
       ensurePermission(uri, Permission.WRITE);
-      if ("file".equals(uri.getScheme())) {
```

Comment:
I assume you removed this check because we can write to SAF urls with the `content://` scheme. Would it be appropriate to check `if ("file".equals(uri.getScheme()) || isSAFUri(getContext(), uri))`? Would be good to still throw a nice error message if someone tries to write to some other unsupported uri (like `asset://...`).

Reply:
great, thanks for the clarification.